src/pages/apply.js

```diff
@@ -0,0 +1,1169 @@
+import React, { useState } from "react"
+import { navigate } from "gatsby"
+import styled from "styled-components"
+import { useToasts } from "react-toast-notifications"
+import Select from "react-select"
+
+import {
+  Form,
+  Label,
+  Input,
+  Checkbox,
+  CheckboxInput,
+  TextArea,
+  Button,
+  Required,
+  Center,
+  H2,
+} from "../components/SharedStyledComponents"
+
+import { colorRed } from "../utils/styles"
+import validateEmail from "../utils/validateEmail"
+import isDev from "../utils/isDev"
+import * as styles from "../utils/styles"
+
+import { COUNTRY_OPTIONS, TIMEZONE_OPTIONS } from "../constants"
+
+const StyledForm = styled(Form)`
+  margin: 2rem auto;
+  max-width: ${styles.screenSizeXL};
+`
+
+const StyledSelect = styled(Select)`
+  margin-top: 0.5rem;
+`
+
+const StyledLabel = styled(Label)`
+  display: ${({ display = true }) => (display ? "flex" : "none")};
+  margin-bottom: 2.5rem;
+`
+
+const ErrorMessage = styled.small`
+  color: ${colorRed};
+`
+
+const ErrorDiv = styled.div`
+  margin-top: 0.5rem;
+  height: 0rem;
+  width: 100%;
+`
+
+const StyledButton = styled(Button)`
+  margin-bottom: 1rem;
+`
+
+const GENDER_TYPES = ["She/Her", "Him/His", "They/Them", "Other"]
+
+const YESNO = ["Yes", "No"]
+
+const PROJECTRESEARCHIDEA = [
```

Comment:
```suggestion
const PROJECT_RESEARCH_IDEA = [
```
Let's continue the format of separating words with underscore. Could also be e.g. `PROJECT_IDEA`.